src/pages/ws/ui/modal-notes.tsx

```diff
@@ -24,44 +25,59 @@ export const ModalNotes = observer(() => {
 
   const currentText = get(toJS(tagsWithNotes), currentTag)
 
-  const [value, setValue] = useState(currentText)
+  const [value, setValue] = useState(currentText || '')
+  const [error, setError] = useState('')
 
   const handleSaveNote = () => {
     variantStore.updateTagsWithNotes([currentTag, value])
     handleClose()
   }
 
+  const handleChange = (note: string) => {
+    validateNotes(note, setError)
+    setValue(note)
+  }
+
   return (
     <ModalBase refer={ref} minHeight={200} width="500px">
       <HeaderModal
         groupName={`${t('variant.notesFor')} [${genInfo}] ${hg19}`}
         handleClose={handleClose}
       />
 
-      <div ref={ref} className="flex flex-col mt-4">
+      <div ref={ref} className="flex flex-col mt-2">
         <div>{currentTag}</div>
 
-        <textarea
-          placeholder="Enter text"
-          className="w-full mt-1 p-3 h-80 rounded-lg resize-none mx-auto shadow-dark"
-          rows={15}
-          value={value}
-          onChange={(e: any) => setValue(e.target.value)}
-        />
+        <div className="relative mt-5">
+          {error && (
+            <div className="absolute -top-3.5 text-12 text-red-secondary">
+              {error}
+            </div>
+          )}
+
+          <textarea
+            placeholder="Enter text"
+            className="w-full mt-1 p-3 h-80 rounded-lg resize-none mx-auto shadow-dark"
+            rows={15}
+            value={value}
+            onChange={(e: any) => handleChange(e.target.value)}
+          />
+        </div>
 
         <div className="w-full flex justify-end ml-auto mt-6">
           <div className="flex items-center">
             <Button
               text={t('general.cancel')}
-              variant={'secondary-dark'}
-              className="border-grey-light hover:bg-grey-light"
+              variant={'secondary'}
+              className="hover:bg-blue-bright hover:text-white"
```

Comment:
The styles are not consistent with the design scheme:
<img width="516" alt="Screenshot 2022-01-18 at 01 13 54" src="https://user-images.githubusercontent.com/96476703/149842520-456c06f6-48db-4bf9-bb45-dfd183cc8157.png">